### DISCUSSION TITLE
Formatting

### DISCUSSION THREAD

**Discussion by 0x-codeX** [Cyfrin/security-and-auditing-full-course-s23#188] [Q&A]
How does Patrick format his VS code to be more vertical than horizontal, the code seems more easier to read vertically.

**Comment by EngrPips** [comment]
Hmmmmm, I don't know how he formats his VS code. @PatrickAlphaC  would maybe help you know when they are available.

**Comment by PatrickAlphaC** [comment]
What do you mean "more vertical than horizontal"?

You might be referring to [word wrap](https://code.visualstudio.com/docs/editor/codebasics#_how-do-i-turn-on-word-wrap).

You can enable this with `option + z` on a mac.
